src/components/EntryForm.js

```diff
@@ -509,12 +522,68 @@ const EntryForm = ({ title, loo, center, children, ...props }) => {
                 );
               })}
             </ol>
-
-            <Spacer mt={4} />
           </div>
 
+          <Spacer mt={4} />
+
+          <h2 id="campaignUOL-heading">
+            7. Do you know if the toilet belongs to the Use Our Loos Scheme?
+          </h2>
+
+          <Spacer mt={2} />
+
+          <Box ml={[2, 4]}>
+            <Box display="flex" alignItems="center">
+              <Controller
+                as={Switch}
+                aria-labelledby="campaignUOL-heading"
+                name="campaignUOL"
+                control={control}
+                valueName="checked"
+                defaultValue={loo.campaignUOL || false}
+              />
+
+              <Spacer ml={2} />
+
+              <Text fontSize={2}>
+                {getValues('campaignUOL') ? 'Yes' : 'No'}
+              </Text>
+            </Box>
+
+            <Spacer mt={3} />
+
+            <Box display="flex" alignItems={['flex-start', 'flex-end']}>
+              <img
+                src={uolLogo}
+                alt="Use Our Loos"
+                css={{
+                  height: '3em',
+                  float: 'left',
+                }}
+              />
+
+              <Spacer ml={3} />
+
+              <Text as="p" fontSize={2}>
+                Use our loos is a national campaign working to unlock the loos
+                that are hidden in high street cafes, restaurants, coffee shops
+                and bars.&nbsp;
+                <Button
```

Comment:
@falconx  Maybe we should send to target=_blank to avoid losing part-filled form data?

Reply:
oh yes, definitely.